modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -321,6 +322,12 @@ export default function ViewObjectDefinitions({
 		};
 	}, []);
 
+	useEffect(() => {
+		if (reloadFDS) {
+			setTimeout(() => setReloadFDS(false), 1000);
```

Comment:
@Artartur, We can reduce this time to 200 milliseconds, it will work in the same way, but giving the user a faster reload impression.

Reply:
Done!